src/cwv/handler.js

```diff
@@ -81,6 +83,9 @@ export async function convertToOppty(auditUrl, auditData, context) {
         'Traffic acquisition',
         'Engagement',
       ],
+      data: {
+        ...kpiDeltas,
```

Comment:
In the else case, when the opportunity already exists but the audit is re-run, we should also update the kpiDeltas in that case

Reply:
Apologies, maybe I misunderstood the context.

I am updating kpiDeltas in both cases: when the opportunity is created and when it is updated.
https://github.com/adobe/spacecat-audit-worker/blob/d0599b397728d3595b1e1f103895dc120d66d207/src/cwv/handler.js#L64

Thank you for the review!

Reply:
Fixed with the commit:
https://github.com/adobe/spacecat-audit-worker/pull/508/commits/d0599b397728d3595b1e1f103895dc120d66d207

Resolving this comment